CMakeLists.txt

```
@@ -720,6 +720,29 @@ IF (ASPECT_UNITY_BUILD)
       SET_PROPERTY(TARGET ${_T} PROPERTY UNITY_BUILD TRUE)
     ENDFOREACH()
     MESSAGE(STATUS "Combining source files into unity build.")
+
+    # If cmake >= 3.18 we can also use unity groups
+    IF (NOT CMAKE_VERSION VERSION_LESS 3.18)
```

Probably easier to read this way:
```suggestion
    IF (CMAKE_VERSION VERSION_GREATER_EQUAL 3.18)
```

---

This is new in cmake 3.7. We'd have to check what the real minimum version is.... ... ah, bummer: https://github.com/dealii/dealii/blob/dealii-9.4/CMakeLists.txt So deal.II 9.4 only required cmake 3.3...